src/button/default.scss

```diff
@@ -27,13 +36,34 @@
     }
     @include iui-notification-marker-hover;
 
-    > .iui-icon {
+    > .iui-button-icon {
       @include themed {
         fill: t(iui-icons-color-actionable-hover);
       }
     }
   }
 
+  &[disabled],
+  &[disabled]:hover,
+  &[disabled]:active,
+  &[disabled]:focus {
+    @include iui-button-disabled;
+    background-color: transparent;
+    border-color: transparent;
+
+    > .iui-button-icon {
+      @include themed {
+        fill: t(iui-icons-color-actionable-disabled);
+      }
+    }
+
+    &.iui-active {
+      @include themed {
+        background-color: rgba(t(iui-color-foreground-body-rgb), 0.05);
+      }
+    }
```

Comment:
I think all of these rules could moved into the `iui-button-disabled` mixin.

I see that these rules should not apply to cta/hv buttons so maybe the mixin could have a parameter.